packages/kotti-ui/source/kotti-comment/components/CommentReply.vue

```diff
@@ -1,19 +1,19 @@
 <template>
```

Comment:
i just realised this entire file is almost a replica of the root file `KtComment` 
anyway we can combine those? 
this file should be the only file with the code, and the `KtComment` component itself needs to consult this file twice